src/arch/mod.rs

```diff
@@ -110,7 +141,7 @@ impl Type {
             // Clang uses the largest alignment of any element as the alignment of the whole
             // Not sure why, but who am I to argue
             // Anyway, Faerie panics if the alignment isn't a power of two so it's probably for the best
-            Union(struct_type) | Struct(struct_type) => struct_align(&struct_type.members()),
+            Union(struct_type) | Struct(struct_type) => struct_type.struct_align(),
```

Comment:
You can probably rename this function to just `align()` now that it's an impl on `StructType`.

Reply:
Ditto for `struct_offset()` -> `offset()`.